chart/modeldb/charts/postgres/templates/NOTES.txt

```diff
@@ -0,0 +1,43 @@
+** Please be patient while the chart is being deployed **
```

Comment:
I'm not sure what "earlier" you are referring to. Charts should not be picked automatically. You need to pass the repository or location you want to use. Using an external one is doable, but I'm not sure it would work with subcharts. So it would be on the user to install the two completely separate systems and connect them, which is not nice.